apps/sms/views/conversation/js/compose.js

```diff
@@ -862,7 +919,83 @@ var Compose = (function() {
       };
 
       return defer.promise;
-    }
+    },
+
+    onSegmentInfoChange: function c_onSegmentInfoChange() {
```

Comment:
Some update here: I gave it a try and drafted out AppNotification module that can create notifications dynamically, hide immediately and with timeout. And I want to say that supporting all our use cases isn't trivial task and requires a bit of refactoring. So having said that, I'm totally fine to not do this in this PR and do later if we need it :)